frame/composable-traits/src/dex.rs

```diff
@@ -100,6 +100,8 @@ pub trait CurveAmm {
 		who: &Self::AccountId,
 		assets: Vec<Self::AssetId>,
 		amplification_coefficient: Self::Balance,
+		fee: Permill,
+		admin_fee: Permill,
```

Comment:
is admin fee vault fee? we should have fee afaik to earn on exchanges, but vault will dictate what to do with that extra money, not admin (like in lending)

Reply:
dex calls vault, not vault call dex

Reply:
well currently I have not thought how to integrate dex and vault. The fee and admin_fee is added for dex to operate independently. For now it it zero in all tests. But yes I agree it may require some change when we work on vault-dex integration.